week-3/extended-project/nowhere/views/about.erb

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>NoWhere | About us</title>
+    <link
+      rel="icon"
+      type="image/png"
+      href="https://i.imgur.com/3t8FYL1.png"
+      sizes="32x32"
+    />
+
+    <!-- Custom-Files -->
+    <link rel="stylesheet" href="css/bootstrap.css" />
+    <!-- Bootstrap-Core-CSS -->
+    <link rel="stylesheet" href="css/style.css" type="text/css" media="all" />
+    <!-- Style-CSS -->
+    <!-- font-awesome-icons -->
+    <link href="css/font-awesome.css" rel="stylesheet" />
+    <!-- //font-awesome-icons -->
+    <!-- /Fonts -->
+    <link
+      href="//fonts.googleapis.com/css?family=Poppins:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900"
+      rel="stylesheet"
+    />
+    <link
+      href="//fonts.googleapis.com/css?family=Source+Sans+Pro:200,200i,300,300i,400,400i,600,600i,700,700i,900"
+      rel="stylesheet"
+    />
+  </head>
+  <body class="menber">
+    <header class="header">
+      <div class="container-fluid px-lg-5">
+        <!-- nav -->
+        <nav class="py-4">
+          <div id="logo">
+            <h1><a href="/">Nowhere</a></h1>
+          </div>
+          <label for="drop" class="toggle">Menu</label>
+          <input type="checkbox" id="drop" />
+          <ul class="menu mt-2">
+            <li><a href="/">Home</a></li>
+            <li class="active"><a href="/about">About</a></li>
+            <li><a href="/team">Team</a></li>
+          </ul>
+        </nav>
+        <!-- //nav -->
+      </div>
+    </header>
+    <div class="container">
+      <section class="hand-crafted bg-light py-5">
+        <div class="container py-xl-5 py-lg-3">
+          <div class="row banner-grids">
+            <div class="col-lg-6 banner-image">
+              <div class="effect-w3">
+                <img
+                  src="https://files.slack.com/files-pri/TEH2PTB37-FGX618TFA/img_20190314_180623.jpg"
+                  alt=""
+                  \

```

Comment:
Why this `\`?

Reply:
Auto completition in VS Code. \ is not required but VS Code just adds it. =|